lib/spack/spack/cmd/arch.py

```diff
@@ -27,6 +27,8 @@
 import spack.architecture as architecture
 
 description = "print architecture information about this machine"
+section = "system"
+level = "short"
```

Comment:
We can always change it later.  Personally I think people want to know what their architecture is.